packages/vite/src/generators/vitest/vitest-generator.ts

```diff
@@ -118,6 +118,12 @@ export async function vitestGeneratorInternal(
   );
   tasks.push(installCoverageProviderTask);
 
+  // Setup workspace config file (https://vitest.dev/guide/workspace.html)
+  tree.write(
```

Comment:
Let's check if it's not a root project and generate the file at that point.

Reply:
Yes yes thanks!